designsafe/apps/projects/models/agave/base.py

```diff
@@ -450,6 +450,13 @@ def generate_licenses(pub):
             "description": pub.project.value.description,
             "license": pub.licenses.works
         })
```

Comment:
Should this clause be deleted since we're removing the rest of the "Share Alike" nomenclature, or will that break legacy publications that used the license?

Reply:
Yeah this is going to break for older publications if we don't go back and update their existing values. That being said, a task was created to revert having multiple licenses on a publication, so we'll probably have to go back and update those fields to get rid of multiple licenses anyways. Maybe we should move these license changes into a separate ticket/branch where we can also include the changes to allow only one license selection per publication.